dev_setup.sh

```diff
@@ -368,6 +368,12 @@ function redhat_install() {
 
 }
 
+
+function gentoo_install() {
+    $SUDO emerge --noreplace dev-vcs/git dev-lang/python dev-python/setuptools dev-python/virtualenv dev-python/pygobject sys-devel/libtool virtual/libffi virtual/jpeg dev-libs/openssl sys-devel/autoconf sys-devel/bison dev-lang/swig dev-libs/glib media-libs/portaudio media-sound/mpg123 media-libs/flac net-misc/curl sci-mathematics/fann sys-devel/gcc app-misc/jq
```

Comment:
is dev-python/virtualenv needed? Mycroft uses the builtin venv module, from a quick look the dev-python/virtualenv package is the thirdparty virtualenv-module. But many moons have passed since I last ran gentoo so I might be wrong :)

Reply:
@forslund thanks I will check that....

Reply:
Did you check this? Is it ready to merge as it is or should the virtualenv package be removed?

Reply:
@forslund thanks for the reminder. I am still on it. I can't check it on my Desktop machine where I started the pull request because I have other dependencies to dev-python/virtualenv that will break my whole system. I started a stage3 gentoo docker container but in docker it takes some time to prepare the environment.